tests/integration/test_integration.py

```diff
@@ -17,8 +20,7 @@
 GRAFANA_AGENT_CHARM_NAME = "grafana-agent-k8s"
 NRF_CHARM_NAME = "sdcore-nrf-k8s"
 NRF_CHARM_CHANNEL = "1.5/edge"
```

Comment:
Not strictly related to this PR, but we should probably update to `1.6/edge`.

Reply:
good catch